docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1104,6 +1106,44 @@ async function toggleTranscription(enable = false){
   }
 }
 
+async function toggleHighlights() {
```

Comment:
Using toggle is an easier way for users to access highlighting.
Here, we check for the button data status and if it is set as 'true', it indicated that it was turned on and hence needs to be _switched off_ - hence the first block is that flow while the second block is the flow for _turning on_ the highlights.